napari/layers/shapes/_shapes_mouse_bindings.py

```diff
@@ -710,22 +710,25 @@ def _move_active_element_under_cursor(
                 )
 
             # prevent box from shrinking below a threshold size
-            size = [
-                np.linalg.norm(box[Box.TOP_CENTER] - c),
-                np.linalg.norm(box[Box.LEFT_CENTER] - c),
-            ]
-            threshold = layer._vertex_size * layer.scale_factor / 2
-            scale[abs(scale * size) < threshold] = 1
+            size = (np.linalg.norm(box[Box.TOP_LEFT] - c),)
```

Comment:
Sort of not related to this PR directly, but can we rename this `c` to something more meaningful?
it's defined as:
`c = box[Box.CENTER]`
so maybe rename to `center_of_box`?

Reply:
fixed :P